db/seeds.rb

```diff
@@ -5,3 +5,9 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+Message.create('Hello')
+Message.create('Hola')
+Message.create('Bonjour')
+Message.create('Ciao')
```

Comment:
- [ ] To fix the error in your code, kindly declare the ntry like this:

### Sample:
```
# db/seeds.rb

Message.create(greeting: "Hello, World!")
```
-- The lack of `greeting` method is the reason of the error